ledger/ledger_test.go

```diff
@@ -1078,3 +1078,137 @@ func TestLedgerReload(t *testing.T) {
 		}
 	}
 }
+
+// TestGetLastCatchpointLabel tests ledger.GetLastCatchpointLabel is returning the correct value.
+func TestGetLastCatchpointLabel(t *testing.T) {
+
+	//initLedger
+	genesisInitState, _ := testGenerateInitState(t, protocol.ConsensusCurrentVersion)
+	const inMem = true
+	log := logging.TestingLog(t)
+	cfg := config.GetDefaultLocal()
+	cfg.Archival = true
+	ledger, err := OpenLedger(log, t.Name(), inMem, genesisInitState, cfg)
+	require.NoError(t, err, "could not open ledger")
+	defer ledger.Close()
+
+	// set some value
+	lastCatchpointLabel := "someCatchpointLabel"
+	ledger.accts.lastCatchpointLabel = lastCatchpointLabel
+
+	// verify the value is returned
+	require.Equal(t, lastCatchpointLabel, ledger.GetLastCatchpointLabel())
+}
+
+// generate at least 3 asset and 3 app creatables, and return the ids
+// of the asset/app with at least 3 elements less or equal.
+func generateCreatables(numElementsPerSegement int) (
+	randomCtbs map[basics.CreatableIndex]modifiedCreatable,
+	assetID3,
+	appID3 basics.CreatableIndex,
+	err error) {
```

Comment:
nit/style: hard to read ') {` on the next line might help:

```
func generateCreatables(numElementsPerSegement int) (
	randomCtbs map[basics.CreatableIndex]modifiedCreatable,
	assetID3,
	appID3 basics.CreatableIndex,
	err error
) {
	_, randomCtbs = randomCreatables(numElementsPerSegement)
...
}
```

Reply:
np

Reply:
will adding a new line help? 
otherwise, will need to add a ',' after error. Not nice.